lib/store.rb

```diff
@@ -0,0 +1,10 @@
+module Store
+  MENU =  [
+    {"name" => "milk", "price" => 3.97, "sale_qun" => 2, "sale_price" => 5 },
+    {"name" => "bread", "price" => 2.17, "sale_qun" => 3, "sale_price" => 6 },
+    {"name" => "banana", "price" => 0.99, "sale_qun" => nil, "sale_price" => nil},
+    {"name" => "apple", "price" => 0.89, "sale_qun" => nil, "sale_price" => nil}
+  ]
+  
+  ITEMNAMES = ["milk","bread","banana","apple"]
```

Comment:
Instead of 2 different arrays we could be using a single object to maintain the integrity of the list.